### PR TITLE
make it rails independent

### LINKED ISSUE #53: Use with Sinatra or Padrino?
I dont think seedbank is currently supported for these. Has it ever been considered? Would you have a few tips on what it might require?

### PR DESCRIPTION
**Pull request by ivanovaleksey**
Hope it will fix #53 

It still works within my Rails 4 app, but I haven't tested it yet with Sinatra or Padrino.
So I think it is just an initial version.

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 Rails.backtrace_cleaner.remove_silencers!
 
-Seedbank.seeds_root = File.expand_path('dummy/db/seeds', __FILE__)
+Seedbank.application_root = Pathname.new(File.expand_path('../dummy', __FILE__))
```
Comment (ivanovaleksey) [inline test/test_helper.rb]: I thought I should prefer single quotes 😄

**Comment by ivanovaleksey** [comment]
I found very simple Sinatra application, remove some stuff and [push it](https://github.com/ivanovaleksey/sinatra-seedbank).
The application uses the gem from my fork.
It seems like it works fine.
I created `db/seeds` directory with dummy files. 
The gem defines the tasks correctly so we can see them by running `rake -T` and execute them via rake.

**Comment by james2m** [comment]
Cool. Id been thinking how to run the tests in different environments so this could be helpful. 

a, pre, code, a:link, body { word-wrap: break-word !important; }

File: test/test_helper.rb (hunk 1 above)
Comment (james2m) [inline test/test_helper.rb]: I definitely prefer single quotes unless interpolation, it's a good signpost to look closer when there are double quotes.

**Comment by ivanovaleksey** [comment]
Hi James, yes, I will try to complete this PR. 

#60 introduces a new littile Rails dependency so rebase would be required.

**Comment by james2m** [comment]
This is manually merged onto master.

**Comment by ivanovaleksey** [comment]
Hi James, thank you!
I am sorry because I didn't have enough time to complete this 😞 
But I definitely will test it once again with simple Sinatra app (which I would share).
What do you mean by "manually merged"? I don't see any changes on master branch related to this.
Am I missing something?

**Comment by james2m** [comment]
Hey @ivanovaleksey thanks for the work, you pushed things forward when I wasn't doing so!

You should see your changes now on master with a few tweaks from the merge. Unfortunately it's late here and Travis is complaining so I'll have to do a little more to fix that, but it's otherwise looking good.

**Comment by ivanovaleksey** [comment]
Now I see the commit. Is there something else I can help you with (besides from testing it with Sinatra app)?

**Comment by james2m** [comment]
If you fancy having a crack at #43 it's easy to re-create but I've not had time to add duplicate checking. I'd like it to bomb out with a polite error when a 2nd seed of the same name is defined. In the meantime I'm going to look at multi environment tests as I should cert against rails 5.

**Comment by james2m** [comment]
Back on topic, I'll leave this open until we have some tests in place for non rails apps.

**Pull request closed without merging**